### PR TITLE
jupyter notebook changes

### PR DESCRIPTION
**Pull request by AliceBalfanz**
The jupyter notebooks in the examples folder are outdated and not working with xcube 0.3.0 anymore. Therefore this PR makes the jupyter notebooks complatible with xucbe 0.3.0 and also points to the correct bucket, where out demo cubes for dcs4cop are located.

### CODE REVIEW

File: examples/notebooks/show_remote_cubes.py [hunk 1]
```diff
@@ -29,10 +30,16 @@ def show_remote_cubes(bucket, endpoint_url, region_name):
...
-                df = df.append({'cube_name': filename,
+                try:
+                    spat_cov = (ds.attrs['geospatial_lon_min'], ds.attrs['geospatial_lat_min'],
+                                ds.attrs['geospatial_lon_max'], ds.attrs['geospatial_lat_max'])
+                except KeyError:
+                    spat_cov = None
+                df = df.append({'cube name': filename,
                                 'chunks (time, lat, lon)': chunksize,
-                                'num_vars': len(var_list),
+                                'number of variables': len(var_list),
```
Comment (forman) [inline examples/notebooks/show_remote_cubes.py]: spaces in field names are never a good idea

File: examples/notebooks/show_remote_cubes.py [hunk 2]
```diff
@@ -29,10 +30,16 @@ def show_remote_cubes(bucket, endpoint_url, region_name):
...
                                 'variables': (str(var_list)).replace('[', '').replace(']', '').replace("'", ""),
-                                'start_date': start_date,
-                                'end_date': end_date}, ignore_index=True)
+                                'start date': start_date,
```
Comment (forman) [inline examples/notebooks/show_remote_cubes.py]: same here.

File: examples/notebooks/show_remote_cubes.py [hunk 3]
```diff
@@ -15,7 +15,8 @@ def show_remote_cubes(bucket, endpoint_url, region_name):
 
     cube_names = []
     df = pd.DataFrame(
-        columns=['cube_name', 'chunksize (time, lat, lon)', 'var_nums', 'variables', 'start_date', 'end_date'])
+        columns=['cube name', 'chunks (time, lat, lon)', 'number of variables', 'variables', 
+                 'start date', 'end date', 'spatial coverage (lon_min, lat_min, lon_max, lat_max)'])
```
Comment (forman) [inline examples/notebooks/show_remote_cubes.py]: spaces in field names are never a good idea

File: examples/notebooks/show_remote_cubes.py [hunk 4]
```diff
@@ -15,7 +15,8 @@ def show_remote_cubes(bucket, endpoint_url, region_name):
 
     cube_names = []
     df = pd.DataFrame(
-        columns=['cube_name', 'chunksize (time, lat, lon)', 'var_nums', 'variables', 'start_date', 'end_date'])
+        columns=['cube_name', 'chunks (time, lat, lon)', 'number_of_variables', 'variables',
```
Comment (forman) [inline examples/notebooks/show_remote_cubes.py]: `'chunks (time, lat, lon)'` still contains spaces, i.e. characters that are not valid for Python identifiers.

File: examples/notebooks/show_remote_cubes.py [hunk 5]
```diff
@@ -29,10 +30,16 @@ def show_remote_cubes(bucket, endpoint_url, region_name):
                 ed = pd.to_datetime(str(ds.time.values[-1]))
                 end_date = ed.strftime('%Y-%m-%d')
                 chunksize = ds[var_list[0]].data.chunksize
+                try:
+                    spat_cov = (ds.attrs['geospatial_lon_min'], ds.attrs['geospatial_lat_min'],
+                                ds.attrs['geospatial_lon_max'], ds.attrs['geospatial_lat_max'])
+                except KeyError:
+                    spat_cov = None
                 df = df.append({'cube_name': filename,
                                 'chunks (time, lat, lon)': chunksize,
```
Comment (forman) [inline examples/notebooks/show_remote_cubes.py]: See comment above.

Unfortunately, there is currently no way to add metadata to `DataFrame` series, e.g. `df.chunks.attrs['description'] = 'chunk sizes (time, lat, lon)'`

**Comment by AliceBalfanz** [comment]
Made changes after talking to forman and merging.

**Pull request merged**